pxr/imaging/rprUsd/contextHelpers.cpp

```diff
@@ -177,6 +177,23 @@ rpr_int GetPluginID(RprUsdPluginType pluginType) {
     return pluginID;
 }
 
+bool CheckCompatibility(const std::string& name) {
+    // the purpouse of this check is to avoid some "virtual" GPUs for witch contect could be created, but crashes when used
+    std::string lowercaseName = name;
+    std::transform(lowercaseName.begin(), lowercaseName.end(), lowercaseName.begin(), [](unsigned char c) { return std::tolower(c); });
+    
+    return (lowercaseName.find("amd") != std::string::npos ||
```

Comment:
Maybe make this check in loop? Like use initializer list of strings and make check on body